### DISCUSSION TITLE
Schedule support difference between spring and jdk

### DISCUSSION THREAD

**Discussion by xiangtianyu** [open-telemetry/opentelemetry-java-instrumentation#11744] [Q&A]
I found that jdk Executor only transform "schedule" while spring schedule only transform "scheduleAtFixedRate" and "scheduleWithFixedDelay"
<img width="844" alt="image" src="https://github.com/open-telemetry/opentelemetry-java-instrumentation/assets/10825900/0874f8fa-ab56-4b0e-b8c4-e880654e665e">
<img width="740" alt="image" src="https://github.com/open-telemetry/opentelemetry-java-instrumentation/assets/10825900/dad7f00c-4970-49f5-a549-006096688fc2">

What cause this difference and why not both transform all these methods?

**Comment by laurit** [comment]
> What cause this difference and why not both transform all these methods?

Probably because these instrumentations do completely different things. If you believe you have found a bug please include a minimal application that reproduces the issue.
